### PR TITLE
test(swift-traykit): make SliccTrayVFS the best-covered Swift package

### PR DESCRIPTION
**Pull request by trieloff**
## Summary

`swift-traykit` was the lowest Swift coverage floor (60/55/58) and the nightly ratchet skipped it because `SWIFT_BUNDLES` had no entry. This PR covers the remaining testable branches, wires the package into the ratchet, and raises floors to **97/93/94** — above `swift-server` (96/91/92).

Measured locally (`swift-coverage-check.sh`):

| Metric | Before floor | Measured | New floor |
| --- | ---: | ---: | ---: |
| Lines | 60 | 98.37% | 97 |
| Functions | 55 | 94.34% | 93 |
| Regions | 58 | 95.11% | 94 |

## What changed

- New tests for FsClient payload/error paths, LeaderVFSProvider mutations/enumerator, File Provider connection pool routing, credential load/save edge cases, and SystemTrayCredentialKeychain.
- Shared XCTest doubles extracted so the existing suite and the new files do not fork FakeFS.
- SWIFT_BUNDLES moved to coverage-ratchet-lib.mjs and a test now fails if a Swift floor has no bundle. swift-traykit and swift-widgetkit are both mapped (the latter was the same skip).
- SystemTrayCredentialKeychain is internal so macOS `swift test` can exercise the iOS keychain helper that still compiles into the binary.

## Verification

- `cd packages/swift-traykit && swiftlint lint && swift format lint --strict --parallel --recursive Sources Tests Package.swift`
- `cd packages/swift-traykit && swift test` — 95 tests, 0 failures
- `./packages/dev-tools/tools/swift-coverage-check.sh packages/swift-traykit SliccTrayVFSPackageTests` — 98.37 / 94.34 / 95.11 vs floors 97 / 93 / 94
- `npx vitest run packages/dev-tools/tools/coverage-ratchet-lib.test.mjs`

### CODE REVIEW

File: packages/swift-traykit/Tests/SliccTrayVFSTests/TrayCredentialStoreTests.swift [hunk 1]
```diff
@@ -178,6 +178,85 @@ final class TrayCredentialStoreTests: XCTestCase {
...
+
+    func testSystemKeychainWriteReadAndClear() {
+        let keychain = SystemTrayCredentialKeychain(
+            accessGroup: "slicc.traykit.tests.\(UUID().uuidString)")
+        let payload = Data("https://tray.example/join/keychain".utf8)
+        _ = keychain.write(payload)
+        _ = keychain.write(payload)
+        _ = keychain.read()
+        keychain.clear()
+        _ = keychain.read()
```
[the remark this answers is not in the document]
Comment (trieloff) [reply packages/swift-traykit/Tests/SliccTrayVFSTests/TrayCredentialStoreTests.swift]: Addressed on `ea8e6761c`.

`testSystemKeychainWriteReadAndClear` no longer discards results. If `write` succeeds it asserts a round-trip, a second write, and `clear` → nil. If the random access group is not an entitlement of `swift test` (the usual macOS case), it asserts the failure contract: read is nil, clear is a no-op, a second write still fails, and read stays nil.

Also declared `WebRTC` on the `SliccTrayVFSTests` target so `lint:swift-deps` matches `TestDoubles.swift`'s `import WebRTC`.
